hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/policy/ContainerChoosingPolicy.java

```diff
@@ -32,5 +33,6 @@ public interface ContainerChoosingPolicy {
    * @return a Container
    */
   ContainerData chooseContainer(OzoneContainer ozoneContainer,
-      HddsVolume volume, Set<Long> inProgressContainerIDs);
+      HddsVolume volume, Set<Long> inProgressContainerIDs,
+      ReplicationSupervisor replicationSupervisor);
```

Comment:
ReplicationSupervisor replicationSupervisor -> Set<Long> replicationContainerIDs

Reply:
Please complete the java doc, adding the parameters, and explain why the two container IDs should be skipped.